target/linux/mediatek/filogic/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac

```diff
@@ -95,6 +95,11 @@ case "$board" in
 	tplink,tl-xdr6088)
 		[ "$PHYNBR" = "0" ] && get_mac_label > /sys${DEVPATH}/macaddress
 		;;
+	zbtlink,zbt-z8103ax)
+		addr=$(mtd_get_mac_binary "Factory" 0x4)
+		[ "$PHYNBR" = "0" ] && macaddr_add $addr 2 > /sys${DEVPATH}/macaddress
+		[ "$PHYNBR" = "1" ] && macaddr_add $addr 3 > /sys${DEVPATH}/macaddress
+		;;
```

Comment:
Is this needed? both PHYs have MAC address assigned via DTS.

Reply:
Hi @raenye, I had to add this again because I can't make the DTS one work without issue.

Reply:
Neither phy worked? or just phy0? I guess the problem is that MT7976CN is a DBDC IC.

Reply:
FYI https://patchwork.kernel.org/project/linux-wireless/list/?series=771380

Reply:
@dangowrt cool! Until upstream accepts, is there a patch to mt76 within OpenWrt?

Also, is this only for mt7976? sounds like mt7615dn could benefit from it too.